### PR TITLE
Update dev env

### PR DESCRIPTION
**Pull request by igordavydsson**
- Ignore data/ and static/ folders in git
- Update webpack-cli version
- Update docker-compose setup

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,19 +3,19 @@ services:
   db:
     container_name: respa-db
     build: ./docker/postgres/
-    restart: always
+    restart: on-failure
     environment:
```
Comment (pkalliok) [inline docker-compose.yml]: I don't understand the ramifications for this.  Is it a developer help to not ditch the db contents upon docker-compose stop & docker-compose up?

File: docker-compose.yml [hunk 2]
```diff
@@ -3,19 +3,19 @@ services:
...
         POSTGRES_USER: respa
         POSTGRES_PASSWORD: respa
         POSTGRES_DB: respa
     volumes:
-      - db_data:/var/lib/postgresql
+      - db_data:/var/lib/postgresql/data
   api:
```
Comment (pkalliok) [inline docker-compose.yml]: This is probably OK, but I'm not sure if it affects anything.

File: docker-compose.yml [hunk 3]
```diff
@@ -3,19 +3,19 @@ services:
...
     container_name: respa-api
     build:
       context: ./
       dockerfile: ./docker/dev/Dockerfile
-    command: bash -c 'tail -f /dev/null'
+    command: python manage.py runserver 0:8000
```
Comment (pkalliok) [inline docker-compose.yml]: Does it really work to just run the server?  README.md says to run the server with a separate command https://github.com/City-of-Helsinki/respa/blob/master/README.md#installation-with-docker .  This is now not in sync with what is said in the README.

File: docker-compose.yml (hunk 2 above)
Comment (pkalliok) [reply docker-compose.yml]: What is not persisted anymore?

Comment (igordavydsson) [reply docker-compose.yml]: without this change every time you stop and start the container, the db data is gone. the container creates a new and empty db.

File: docker-compose.yml (hunk 1 above)
Comment (igordavydsson) [reply docker-compose.yml]: this is not about the contents, this is about the container that does not shut down ever, unless you manually locally change the restart policy, which is super annoying

Comment (igordavydsson) [reply docker-compose.yml]: also this restart policy starts the container automatically upon the launch of docker on the machine, which leads to the port being occupied if you then decide to start another project, where db container uses the same port

File: docker-compose.yml (hunk 3 above)
Comment (igordavydsson) [reply docker-compose.yml]: true, gets out of sync with the README.

Comment (igordavydsson) [reply docker-compose.yml]: updated README 🙂

**Comment by jussih** [comment]
@jussiarpalahti should take a look at the docker-compose changes and see if they affect the automatic deploys to the container-driven test environment.

**Pull request merged**